index.js

```diff
@@ -1,6 +1,43 @@
 module.exports = {
-    gltfPipeline : require('./lib/gltfPipeline'),
+    AccessorReader : require('./lib/AccessorReader'),
```

Comment:
In there a reason we just don't just `module.exports = require('./lib');` here instead of requiring every object individually?